.github/workflows/abidiff.yml

```diff
@@ -19,12 +19,12 @@ jobs:
       - uses: actions/checkout@v3
         with:
           repository: aws/aws-lc
-          ref: ${{ github.ref }}
+          ref: ${{ github.event_name == 'push' && github.event.ref || github.event.pull_request.head.sha }}
```

Comment:
NP: Are parenthesis needed here and below?
```
ref: ${{ (github.event_name == 'push' && github.event.ref) || github.event.pull_request.head.sha }}
```

Reply:
GitHub doesn't seem to outright state precedence order for evaluation, but it appears to be left-to-right. See the example here that is basically the same as the line I have: https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/evaluate-expressions-in-workflows-and-actions#example